### PR TITLE
Willys/381916-coe

### PR DESCRIPTION
**Pull request by wpschaub**
Releasing the first DRAFT of the CoE overview post to trigger open the feedback floodgates and to collaborate on a consistent format to use for the other event posts.

### CODE REVIEW

**Review by adichourasiya — DISMISSED** [review]

**Review by wsbctechnicalblog — CHANGES_REQUESTED** [review]
So happy to see you've placed strong emphasis on the WHY! Without broadcasting clear, crisp expectations CoE stands less chances of moving the needle. Great way to formulate those.

Few minor requests: on the line 47, the last sentence "and products. content marketing engine for your organization" reads awkwardly. There is a full stop, but the word 'content' is not capitalized. Also, I'm not clear on what is the intention of that sentence? Maybe consider reworking it?

On line 69 there is a typo: hypothesist. Please fix it.

**Comment by wpschaub** [comment]
@wsbctechnicalblog thank you for the feedback. I removed the sentence fragment ... must be a late-night brain fart that I could not figure out either. Also fixed the typo. Switching to working group next.

File: content/ceremony-center-of-enablement.md
Comment (sben101) [inline content/ceremony-center-of-enablement.md:93]: Should we mention governance? Who will be chairing these CoE meetings, etc.?

**Review by sben101 — COMMENTED** [review]
Looks great @wsbctechnicalblog I have added my one comment around governance/ chairing of these meetings and not sure if that is something we need to spell out?

File: content/ceremony-center-of-enablement.md
Comment (wpschaub) [reply content/ceremony-center-of-enablement.md:93]: Governance is mentioned. We can add who will be chairing the CoE meetings, but as it is a closed, by invitation-only, event  I am thinking that the team can be self-organising and rotate the chair?

Comment (wsbctechnicalblog) [reply content/ceremony-center-of-enablement.md:93]: I'd leave that to the self-organizing forces within the Center of Enablement. Maybe they elect one member to chair the meetings, maybe they agree to rotate.

**Review by adichourasiya — DISMISSED** [review]

**Review by sben101 — DISMISSED** [review]
Thanks for the updates. This is looking good with the necessary information.

**Comment by wpschaub** [comment]
I have included a "lifetime" section. Hope you like :)

**Review by wsbctechnicalblog — DISMISSED** [review]

**Review by wsbctechnicalblog — DISMISSED** [review]

**Review by wsbctechnicalblog — DISMISSED** [review]
Fixed the typo on line 45.

**Review by AndreKaminski — COMMENTED** [review]
Looks good to me, thank you

**Comment by wpschaub** [comment]
@wsbctechnicalblog I am done with this one. When we have two reviewer approvals and it is Monday, you can launch.

**Comment by wpschaub** [comment]
@AndreKaminski @sben101 @asiebold69 @wsbctechnicalblog @adichourasiya please review this final PR to give it a final check-over. We are planning to trigger the wave of ceremony blogs and dogfood our (1) understanding of events, (2) overview, and working agreement posters, and (3) the supporting SharePoint site.

Here is link to view of blog post: https://github.com/wsbctechnicalblog/wsbctechnicalblog.github.io/blob/de4a10bca28dc79e7db674c952159eb82a26b78f/content/ceremony-center-of-enablement.md

Tic toc plan: 

- 11/8 Publish CoE blog
- 11/9 Working group blog
- 11/10 CoP blog
- 11/11 Dojo blog
- 11/12 Townhall blog
- 11/13 Overview blog

**Comment by wpschaub** [comment]
@AndreKaminski @sben101 @asiebold69 @wsbctechnicalblog @adichourasiya please review this final PR to give it a final check-over. We are planning to trigger the wave of ceremony blogs and dogfood our (1) understanding of events, (2) overview, and working agreement posters, and (3) the supporting SharePoint site.

Here is [link](https://github.com/wsbctechnicalblog/wsbctechnicalblog.github.io/blob/de4a10bca28dc79e7db674c952159eb82a26b78f/content/ceremony-center-of-enablement.md) to quick view of blog post

Tic toc plan: 

- 11/8 Publish CoE blog
- 11/9 Working group blog
- 11/10 CoP blog
- 11/11 Dojo blog
- 11/12 Townhall blog
- 11/13 Overview blog

**Comment by wpschaub** [comment]
@AndreKaminski @sben101 @asiebold69 @wsbctechnicalblog @adichourasiya please review this final PR to give it a final check-over. We are planning to trigger the wave of ceremony blogs and dogfood our (1) understanding of events, (2) overview, and working agreement posters, and (3) the supporting SharePoint site.

Here is link to view of blog post: https://github.com/wsbctechnicalblog/wsbctechnicalblog.github.io/blob/de4a10bca28dc79e7db674c952159eb82a26b78f/content/ceremony-center-of-enablement.md

Tic toc plan: 

- 11/8 Publish CoE blog
- 11/9 Working group blog
- 11/10 CoP blog
- 11/11 Dojo blog
- 11/12 Townhall blog
- 11/13 Overview blog

**Comment by AndreKaminski** [comment]
@wpschaub in the diagram "Why we need CoE", could you make one of the CoE as "...", otherwise people might think that these are the only CoEs, but we have many more. Also I would suggest adding arrow between Working Group and CoP, as the outcomes of WG could go to both CoE and CoP

**Comment by wpschaub** [comment]
@AndreKaminski I have revised the image. Looking for your review approve, to push this onto the launch pad.

**Review by AndreKaminski — DISMISSED** [review]

**Review by wsbctechnicalblog — DISMISSED** [review]

**Pull request merged**
